### DISCUSSION TITLE
New docs text width

### DISCUSSION THREAD

**Discussion by PhilGarb** [TanStack/query#3877] [General]
Hey there,

excellent v4 update. I am just wondering whether there is a reason the new docs let the content area stretch the entire width? In my opinion it makes the content much harder to read compared to the previous docs, especially on larger screens. 

See: 

![CleanShot 2022-07-20 at 10 36 45](https://user-images.githubusercontent.com/38015558/179937124-0a3cddb4-13c1-4f39-b9b0-0a1630ce04b1.png)

compared to:

![CleanShot 2022-07-20 at 10 36 33](https://user-images.githubusercontent.com/38015558/179937147-17710668-a1b1-4754-b7ff-bf1dc0760f2f.png)

which simply adds a max-width and justifies the content. (The overlapping navigation at the bottom is not caused by this. There just happened to be no text in the large version.)

**Comment by TkDodo** [comment]
This is more a question to the tanstack.com repo because that's where the docs are rendered for all of the TanStack: https://github.com/TanStack/tanstack.com

**Reply by PhilGarb** [reply]
Ahh ok thank you. I will ask there.
